Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM ubuntu:14.04
 
-ENV GOVERSION 1.5.3
-
 RUN apt-get update -y && apt-get install --no-install-recommends -y -q \
                          curl build-essential ca-certificates git mercurial bzr \
                       && rm -rf /var/lib/apt/lists/*
 
+ENV GOVERSION 1.5.4
```

Comment:
That's why I moved it below the long running apt-get commands; the only things between where it is now and where it's used are just ENV and directory commands. Are those worth optimizing the reuse of?